frb_codegen/src/generator/dart/ty_struct.rs

```diff
@@ -192,16 +192,31 @@ impl TypeDartGeneratorTrait for TypeStructRefGenerator<'_> {
                 "{}{}class {} {{
                 {}
 
-                {}({{{}}});
+                {}{}({}{}{});
 
                 {}
             }}",
                 comments,
                 metadata,
                 self.ir.name,
+                if constructor_params.is_empty() {
+                    "const ".to_string()
+                } else {
+                    "".to_string()
+                },
```

Comment:
uber-nit:

```suggestion
                (if constructor_params.is_empty() {    "const "  } else {     ""   }).to_string(),
```

similar for the other two below